src/move/emojicoin_dot_fun/sources/emojicoin_dot_fun.move

```diff
@@ -326,13 +383,17 @@ module emojicoin_dot_fun::emojicoin_dot_fun {
             integrator_fee_rate_bps,
             market_ref_mut,
         );
-        let quote;
-        aptos_account::create_account(swapper_address); // For better feedback if no account yet.
 
-        // Prepare variables relevant to global amounts, as they are used in branching logic too.
+        // Prepare local variables.
+        let quote;
         let registry_ref_mut = borrow_registry_ref_mut();
         let quote_volume_as_u128 = (event.quote_volume as u128);
-        let total_quote_locked_ref_mut = &mut registry_ref_mut.total_quote_locked;
+        let global_stats_ref_mut = &mut registry_ref_mut.global_stats;
+        let total_quote_locked_ref_mut = &mut global_stats_ref_mut.total_quote_locked;
+        let market_cap_ref_mut = &mut global_stats_ref_mut.market_cap;
+        let fdv_ref_mut = &mut global_stats_ref_mut.fully_diluted_value;
+
+        aptos_account::create_account(swapper_address); // For better feedback if no account yet.
```

Comment:
I assume we should change this in whichever way we decide to go about it [in the other PR](https://github.com/econia-labs/emojicoin-dot-fun/pull/22/files#r1578744800)